src/cmd/cgogen.go

```diff
@@ -155,11 +154,8 @@ func doGoFile() {
 	fast, err := parser.ParseFile(fset, "", fo, parser.AllErrors|parser.ParseComments)
 	check(err)
 
-	packagePath := ""
-	if get_package_path_from_file_name {
```

Comment:
why delete this ? why not better make `get_package_path_from_file_name` a command line option ?

Reply:
because it will always be `true` always a module comes from one or several files, and also after go 1.11.x that almost manages it only with the` format`. So I see it unnecessary